app/service/hotel_service.py

```diff
@@ -0,0 +1,47 @@
+from app.database import db
+from app.util import formulate_date, get_days_between, add_days
+
+
+def read_hotel(checkInDate: str, checkOutDate: str, destination: str):
+    checkIn_datetime = formulate_date(checkInDate)
```

Comment:
Try to stick to snake case, snake case is usually the convention for python apps 👍🏻